core/runtime.rs

```diff
@@ -565,6 +522,43 @@ impl JsRuntime {
     js_runtime
   }
 
+  /// Create a new [`v8::OwnedIsolate`] and its global [`v8::Context`] from optional parameters and snapshot.
+  fn create_isolate(
```

Comment:
I suggest to merge `JsRuntime::setup_isolate` into this method. There's no point in having two separate functions that act similar

Reply:
Ah, yes that's a great idea. Done.